composer.json

```diff
@@ -25,7 +25,7 @@
   },
   "autoload-dev": {
     "psr-4": {
-      "Kleemans\\Test\\": "test/"
+      "Kleemans\\Tests\\": "tests/"
```

Comment:
Do you have a particular reason why you changed `/test` to `/tests`?
`/src` is also singular, so at first I don't exactly understand that change.

Reply:
we have different types of tests inside `/tests` folder, so it should be plural. and to add one more point, I have never seen any package or framework which has singular `test` folder for the tests.

Reply:
Ah i see. I think the point about other libraries is valid. So renaming to `/tests` is good idea 👍
I don't see much value in putting `AttributeEventsTest.php` inside a `/Unit` subfolder. Could you revert that?

Reply:
okay